packages/testing/config/jest/web/RedwoodWebJestEnv.js

```diff
@@ -0,0 +1,14 @@
+const Environment = require('jest-environment-jsdom')
+
+// Due to issue: https://github.com/jsdom/jsdom/issues/2524
+// Fix from: https://github.com/jsdom/jsdom/issues/2524#issuecomment-736672511
```

Comment:
@dac09 By the way, like discussed in the linked issue, this happens because we want the environment to be `jsdom` but within `globalSetup` we do type-gen or something that is dependent on Node's API. So far, this is working okay as a workaround but if you know how we can separate the environment for jest's `globalSetup` and others that would be an actual fix.